extensions/iidm/src/ThreeWindingsTransformerPhaseAngleClock.cpp

```diff
@@ -9,6 +9,8 @@
 
 #include <powsybl/AssertionError.hpp>
 #include <powsybl/PowsyblException.hpp>
+#include <powsybl/iidm/PhaseTapChanger.hpp>
+#include <powsybl/iidm/RatioTapChanger.hpp>
```

Comment:
ThreeWindingsTransformerPhaseAngleClock.cpp includes `ThreeWindingsTransformer.hpp` and ThreeWindingsTransformer::Leg has `std::unique_ptr<PhaseTapChanger>` and `std::unique_ptr<RatioTapChanger>` members. 
Compilation fails if the definition of PhaseTapChanger is unknown by compiling ThreeWindingsTransformerPhaseAngleClock.cpp : 
```
[ 95%] Building CXX object extensions/iidm/CMakeFiles/ext-iidm.dir/src/ThreeWindingsTransformerPhaseAngleClock.cpp.obj
ThreeWindingsTransformerPhaseAngleClock.cpp
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\memory(1764): error C2027: utilisation du type non d‚fini 'powsybl::iidm::RatioTapChanger'
```

Note : Theses include cannot be added in ThreeWindingsTransformer.hpp because of circular include dependency between PhaseTapChanger/TapChanger/VoltageLevel/ThreeWindingsTransformer